src/index.js

```diff
@@ -11,3 +11,9 @@ function sayHello() {
 function print(textString) {
     console.log(textString);
 }
+
+/**
+ * Write a function that joins two strings
+ */
+
+function newFunction() {}
```

Comment:
Try to have fewer comments in your code, but always try to make the codebase readable, and comprehensible by giving meaningful names to the functions and variables. In this scenario, you could name the function to be something like `joinTextStrings` so that it tells the reader what it does.

Also keep in mind that you shouldn't be too specific either. For example, if you name the function to be `joinTwoTextStrings`, then if you have to extend this functionality to join three, then it makes things complex.

Once you switch to TypeScript, using good-practices will become even easier. That would help you to write something like the following snippet:

```ts
/**
 * Takes an array of strings, joins them using " " characters between each,
 * and returns the result as a string.
 */
function joinTextStrings(textStrings: string[]): string {
  return textStrings.join(" ");
}
```